0011-command-execution-protocol/README.md

```diff
@@ -0,0 +1,170 @@
+---
+name: Command Execution Protocol
+shortname: 11/CEP
+status: raw
+editor: Alexey Shmalko <ashmalko@kaaiot.io>
+---
+
+<!-- toc -->
+
+# Introduction
+Command Execution Protocol is an endpoint-aware 1/KP extension. It is designed to allow command execution on endpoints.
```

Comment:
I'd like to initiate a discussion about that rule as I don't see much value in following it.

The main and the only argument for the rule is avoiding merge conflicts, so here are some more points to the discussion.

Pros:
- Easier to review and left comments as there is no ambiguity which line the comment refers to.

Cons:
- I feel that merge conflicts should be rare for docs as they are not edited that much.
- Trivial conflicts even on a single line are easily resolved with tools. (e.g., kdiff3 is extremely good at that.)
- Merge conflicts are good for docs as they force the merger to re-read a paragraph and ensure it is still consistent.
- Forcing line breaks make it a little harder to properly format nesting structures (i.e., lists), and they look worse in code.

What do you think?

/cc @hunch7